infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -133,9 +158,35 @@ export class AtatWebApiStack extends cdk.Stack {
         tracingEnabled: true,
       },
     }).restApi;
+    this.addEmailRoutes(props);
     this.addTaskOrderRoutes(props);
   }
 
+  private addEmailRoutes(props: AtatWebApiStackProps) {
+    this.functions.push(
+      new ApiSQSFunction(this, "SubmitEmails", {
+        queue: this.emailQueue,
+        lambdaVpc: props.vpc,
+        method: HttpMethod.POST,
+        handlerPath: this.determineApiHandlerPath("submitEmails", "emails/"),
+      }).fn,
+      new ApiSQSFunction(this, "SendEmails", {
+        queue: this.emailQueue,
+        lambdaVpc: props.vpc,
+        method: HttpMethod.GET,
+        handlerPath: this.determineApiHandlerPath("subscribeSendEmails", "emails/"),
+        functionPropsOverride: {
+          deadLetterQueue: this.emailDeadLetterQueue,
+          // ? set reserved concurrency to zero?
+          reservedConcurrentExecutions: 0,
```

Comment:
We probably just don't want to specify this field at all, right? What was the reason for this more specifically? Just trying to understand the context.

Reply:
Yeah, I didn't have it originally, but came across something in the docs about setting it to zero (and I can't find it now...). If I remember correctly it was related to errors in the lambda and not having it be retried. I think I was thinking along the lines of if we wanted errors to go straight to the DLQ, the lambda fails with no retries.

It can be removed though.